### PR TITLE
Add handling of readiness annotation

### PR DESCRIPTION
**Pull request by weimerb**

### CODE REVIEW

File: src/property/FaceliftProperty.h [hunk 1]
```diff
@@ -77,6 +85,19 @@ class PropertyBase
         return m_ownerSignal;
     }
 
+    bool &readiness()
```
Comment (jacky309) [inline src/property/FaceliftProperty.h]: I feel like isReady() would be a better name

File: src/property/FaceliftProperty.h [hunk 2]
```diff
@@ -77,6 +85,19 @@ class PropertyBase
...
+    {
+        return m_readiness;
+    }
+
+    void setReadiness(bool ready)
```
Comment (jacky309) [inline src/property/FaceliftProperty.h]: setReady ?

File: src/property/FaceliftProperty.h (hunk 1 above)
Comment (jacky309) [reply src/property/FaceliftProperty.h]: and it should be const

Comment (weimerb) [reply src/property/FaceliftProperty.h]: That might be rather confusing, if in all other places (even as annotation) we use "readiness" and only here "isReady". I tend to leave it as it is.

A non-const reference is returned here, so the function cannot be made const.

**Pull request closed without merging**